tests/test_hist.py

```diff
@@ -1,17 +1,33 @@
-# import pytest
-# import numpy as np
-from altairexpress import altairexpress as ae
+from altairexpress import hist as ae
 from gapminder import gapminder
+from pytest import raises
 
-# define testing data
-gapminder
 
+def test_hist():
+    # define testing data
+    gapminder
```

Comment:
Could alternatively add. a

```suggestion
@pytest.fixture
```